plugin-gradle/README.md

```diff
@@ -288,6 +288,19 @@ spotless {
     ktfmt('0.15').dropboxStyle() // version and dropbox style are optional
 ```
 
+<a name="applying-ktlint-to-kotlin-files"></a>
+
+### diktat
+
+[homepage](https://github.com/cqfn/diKTat). [changelog](https://github.com/cqfn/diKTat/releases).  You can provide them manually as `configPath`.
+
+```kotlin
+spotless {
+  kotlin {
+    // version and configPath are both optional
+    ktlint('0.4.0').configPath("***/**/diktat-analysis.yml")
```

Comment:
Why asterisks? Why three asterisks?

Reply:
It's better to write something like `"full/path/to/diktat-analysis.yml"`, because `**` has a very definite meaning.